sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveUtilsSuite.scala

```diff
@@ -42,4 +47,41 @@ class HiveUtilsSuite extends QueryTest with SQLTestUtils with TestHiveSingleton
       assert(hiveConf("foo") === "bar")
     }
   }
+
+  test("ChildFirstURLClassLoader's parent is null") {
```

Comment:
what is this test trying to test?

Reply:
just indicates that if we don't handle child first classloader's parent, we will get caught in exception. if it bothers, i can del it

Reply:
I think we can remove it. The below tests case would fail without your PR, that proves your patch already.